internal/workflow_testsuite.go

```diff
@@ -159,6 +160,12 @@ func (s *WorkflowTestSuite) SetContextPropagators(ctxProps []ContextPropagator)
 	s.contextPropagators = ctxProps
 }
 
+// SetFailureConverter sets the failure converter for this WorkflowTestSuite. If you don't set the failure converter,
+// the default failure converter will be used.
+func (s *WorkflowTestSuite) SetFailureConverter(failureConverter converter.FailureConverter) {
```

Comment:
I wonder if this should be on the activity/workflow environment instead (akin to data converter). @Quinn-With-Two-Ns - thoughts?

Reply:
Yeah I think we should be consistent with how we set the data converter.

Reply:
Makes sense! Added here: https://github.com/temporalio/sdk-go/pull/1484 (and I'll close this PR).